### DISCUSSION TITLE
"Unable to resolve host (freeDNS domain): No address associated with hostname"

### DISCUSSION THREAD

**Discussion by Rein11** [NightscoutFoundation/xDrip#2801] [Q&A] [answered]
Hi all, I don't know if this is the right place to ask, if not, please guide me to a better place :)

I used to have nightscout running for my son on Heroku with a 640G pump/Guardian 3 sensor. I migrated to Fly a couple of months ago and everything seemed to be running fine. Until a couple of days ago i got the message in the 600 series uploader that "nightscout site is unavailable". I didn't know what to do, so I tried to update Fly with using the online guide with Windows Powershell, but i got some errors and i think i might have messed that up. So... I thought I'd start from scratch using the Google Cloud method described by the xDrip team. https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html
Everything seems to be OK, but now I'm at step 5 "Uploading to Nightscout" and I'm not sure what to do next...
I tried to enter the domain created in freeDNS in the 600 series uploader app (secret is the same), and get the following messages:  "nightscout site is unavailable" and "Unable to resolve host (freeDNS domain): No address associated with hostname".  Any idea what i need to do or check next?

**Comment by Navid200** [comment]
Let's do a test first.
What happens if you enter the hostname in a web browser?

If you are not sure what the hostname is, you can find it by choosing the appropriate option at the bottom of the Google Cloud Nightscout status page.
It shows you the hostname and api_secret.

**Reply by Rein11** [reply]
Thanks for the quick response!

I believe I was able to reach it when i was setting it up, it asked me for the API secret. But now it's giving me a "(NAMEOFSUBDOMAIN).cyclone.net’s DNS address could not be found."

**Comment by Rein11** [comment]
![status1](https://user-images.githubusercontent.com/33394545/234288742-e8be107f-a1ba-4356-b8ad-a5db79ddfd3e.png)

Here's an image of the status page, if that helps.

**Reply by Navid200** [reply]
Please note that if you cannot reach your site in a browser, your uploader will not be able to upload to it either.

Please restart the server and try again reaching it in a browser.
https://navid200.github.io/xDrip/docs/Nightscout/Restart.html

**Reply by Rein11** [reply]
That did the trick! Nightscout site asked for the API secret and I can access it again.
The 600 series uploader still gives the same 2 errors though.

**Reply by Rein11** [reply]
I can reach the site from my desktop on different browsers, but on android phones I get a DNS error.

**Reply by Navid200** [reply]
Can you post a screenshot of the DNS error.  Make sure to blackout the hostname if shown.

**Comment by Rein11** [comment]
![WhatsApp Image 2023-04-25 at 16 12 17](https://user-images.githubusercontent.com/33394545/234304822-114a0cbc-91f7-4f0b-a032-38540b57b1dc.jpeg)
![WhatsApp Image 2023-04-25 at 16 12 17 (1)](https://user-images.githubusercontent.com/33394545/234304826-b17e073f-e70c-4017-b3a9-f7d34be9cd6c.jpeg)

**Reply by Navid200** [reply]
Would you get the same error if you temporarily disable WiFi on your phone forcing it to use data?

**Reply by Rein11** [reply]
Same error.

**Reply by Rein11** [reply]
Just asked someone to try it from an iPhone, that seems to work.

**Reply by Navid200** [reply]
Is there also another Android phone to test?

**Reply by Navid200** [reply]
Can you access anything else using that phone?  I mean are you sure it has access to the internet?

**Reply by Navid200** [reply, marked as answer]
Also, please make sure you have no typo in the hostname on the phone.

**Reply by Rein11** [reply]
Hahaha, damnit, I've been an idiot...

I read the domain as cyClone.net, but is was in fact... cylone.net...
So it worked every time i copied the hostname instead of typing it myself...

Well, apologies for taking up your time, and thanks for helping me through this.
And thanks especially for making this awesome tutorial, it really means a lot for us.

Have a great day!

**Reply by Navid200** [reply]
You're welcome and thanks for the feedback
